woocommerce/woocommerce-android
# Issue/3321 toolbar expand on rotation

Fixes #3321 - To test:

* Go to the product list and scroll down so the toolbar collapses
* Tap a product to open product detail
* Rotate the device
* Return to product list and ensure toolbar remains collapsed

Update release notes:
